packages/fields/README.md

```diff
@@ -98,6 +98,10 @@ Undocumented declaration.
 
 Status field for BasePost.
 
+### templateField
+
+Undocumented declaration.
```

Comment:
We should aim to document every export. It's quite discouraging to look at docs that say "Undocumented declaration".